src/Platforms/SQLitePlatform.php

```diff
@@ -861,7 +861,23 @@ private function getForeignKeysInAlteredTable(TableDiff $diff): array
     {
         $oldTable    = $diff->getOldTable();
         $foreignKeys = $oldTable->getForeignKeys();
-        $nameMap     = $this->getDiffColumnNameMap($diff);
+        $keysByName  = [];
+        foreach ($foreignKeys as $key => $foreignKey) {
```

Comment:
Instead of introducing a new loop, `$keysByName` could be populated in the existing one.